Evaluator.java

```diff
@@ -1,8 +1,19 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class Evaluator {
+    private static final Pattern extractEquation = Pattern.compile("(-?\\d+) *([+\\-*/]) *(-?\\d+)");
```

Comment:
Honestly I have no idea if this will work. Let's try it and test it. Testing in Production is fun!